### DISCUSSION TITLE
预告-大学沙龙 李筠-重新理解中世纪——《中世纪》新书分享研讨2023年11月12日 周日早上8点30

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#24] [Announcements]
ZT 大学沙龙（原哈佛沙龙）第194期
 李筠:重新理解中世纪——《中世纪》新书分享与研讨

 
■ 时间与报名信息
 
美东时间：2023年11月11日 周六晚上7点30
北京时间：2023年11月12日 周日早上8点30
 
zoom 报名链接
 https://clt1245285.benchurl.com/c/l?u=1057ED1D&e=1722DC7&c=130065&t=0&l=8266D822&email=McVwm7sr0tzNK29%2BwVCgRXnlkRjLUbPe&seq=1

■ 讲座内容
 
中世纪不是一片黑暗，而是多姿多彩，它最重要的意义在于它是现代世界得以诞生的母体。研究中世纪的核心在于如何认识中世纪与现代世界的系统性关联。《中世纪》从“印象”、“帝国”、“教会”、“王国”、“大学”、“城市”六个方面展开，呈现中世纪演化的复杂脉络，探寻现代世界孕育的过程和逻辑。
 
■ 演讲嘉宾
 
主讲人:李筠
中国政法大学政治学博士、教授、博士生导师、校学术委员会委员，美国哥伦比亚大学访问学者（2015-2016学年），主要研究领域为政治理论、西方政治思想史、中西政治史，著有《论西方中世纪王权观》《英国政治思想新论》《西方史纲》《罗马史纲》《中世纪》。
 
评议人: 周濂
香港中文大学哲学博士，中国人民大学哲学院教授、外国哲学教研室主任，中国人民大学人文杰出青年学者，牛津大学哲学系访问学者、哈佛燕京访问学者。著有《现代政治的正当性基础》（2008）、《你永远都无法叫醒一个装睡的人》（2012）、《正义的可能》（2015），《正义与幸福》（2018），《打开：周濂的100堂西方哲学课》（2019），主编《西方政治哲学史·第三卷》（2017），译有《苏格拉底》《海德格、哲学、纳粹主义》（均为合译），另在各种学术期刊发表中英文论文40多篇。研究领域为政治哲学、道德哲学和语言哲学。
 
主持人：翟志勇
清华大学法学博士，北京航空航天大学法学院教授，哈佛大学访问学者（2008-2009学年），著有《公法的法理学》《从共同纲领到八二宪法》。
 
请注意：届时已报名观众仍需登录Zoom认证以进入本讲座，请确保报名邮箱与Zoom登录邮箱一致，并提前做好相应准备。
大学沙龙的微信公众号被永久封号，请关注沙龙邮件以及哈佛大学费正清中心全球交流网络，获取沙龙最新消息。收看往期讲座请搜索“大学沙龙”YouTube频道。感谢对大学沙龙的支持。
权责声明：
● 嘉宾言论不代表大学沙龙观点和立场。
● 讲座音频、视频、文字等资料版权归大学沙龙所有。未经许可任何个人、媒体不得编辑、传播。
● 媒体合作请联系universityforum1709@gmail.com

关于我们：
大学沙龙是由几名来自中国的哈佛大学访问学者发起的公益学术沙龙。立足哈佛，思考中国与世界。沙龙追求真知，秉持兼容并包，独立开放的精神。

**Comment by yibingbingwu** [comment]
没及时看到此贴，可惜错过了。我本人对中世纪历史也挺感兴趣，觉得是当代社会的雏型和解释。

**Reply by Lewisrxliu** [reply]
不可惜。 可能只是新书简介，在这个分享中，作者并没有阐述很多具体的内容。
